test/extended/quota/clusterquota.go

```diff
@@ -26,6 +26,10 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+// This should be greater than the timeout value in quota reconciliation controller
```

Comment:
```suggestion
// This should be greater than the monitor's sync timeout value in cluster quota reconciliation controller (cluster-policy-controller), because it can hang for 30 seconds when a rare deadlock occurs 
```